server/api/user/index.ts

```diff
@@ -0,0 +1,43 @@
+// GET /api/user
+// endpoint for fetching user data from database
+
+export default defineEventHandler(async (event) => {
+  const { decodedToken, user } = event.context
+
+  // user is not authenticated
+  if (!decodedToken)
+    throw createError({
+      statusCode: 401,
+      statusMessage: 'User not authenticated',
+    })
+
+  if (!user)
+    throw createError({
+      statusCode: 404,
+      statusMessage: 'User data not found',
+    })
+
+  // get scope from query params
+  const { scope } = getQuery(event)
```

Comment:
Samme som med readBody. Er disse built-in? Er ikke vant med å ha så beleilige composables tilgjengelig "out of the box"

Reply:
Det må jeg sjekke ut. Har ikke noe særlig erfaring med context og slikt